circuit_knitting_toolbox/circuit_cutting/__init__.py

```diff
@@ -25,4 +25,8 @@
     cutqc.cut_circuit_wires
     cutqc.evaluate_subcircuits
     cutqc.reconstruct_full_distribution
+    partition_circuit_qubits
 """
+from .cutting_decomposition import partition_circuit_qubits
+
+__all__ = ["partition_circuit_qubits"]
```

Comment:
probably worth doing sooner rather than later
```suggestion
__all__ = [
    "partition_circuit_qubits",
]
```